turbopack/crates/turbopack-core/src/output.rs

```diff
@@ -20,6 +20,10 @@ pub trait OutputAsset: Asset {
     fn references(self: Vc<Self>) -> Vc<OutputAssets> {
         OutputAssets::empty()
     }
+
+    fn len(self: Vc<Self>) -> Vc<Option<u64>> {
+        Vc::cell(None)
+    }
```

Comment:
This is bikeshedding, but I don't love the name `len` here because it's a bit ambiguous about what it means, and `len` usually implies a return type of `usize` (but I agree that u64 probably makes more sense).

What about `size_bytes`?